src/commands/timeout.ts

```diff
@@ -0,0 +1,162 @@
+                    return interaction.reply({ content: "Aww, please don't time out yourself! 💖", ephemeral: true });
+                }
+
+                if (member.moderatable === false) {
+                    return interaction.reply({
+                        content:
+                            '❌ I cannot time out this user! \n**Please make sure that my highest role is above theirs.**',
+                        ephemeral: true,
+                    });
+                }
+
+                const duration = interaction.options.getString('duration', true);
+                const msDuration = ms(duration);
+
+                if (msDuration == null) {
+                    return interaction.reply({
+                        content: '❌ Please specify a valid timeout duration.',
+                        ephemeral: true,
+                    });
+                }
+
+                if (msDuration <= 0) {
+                    return interaction.reply({
+                        content: '❌ The duration of the timeout must be greater than 0!',
+                        ephemeral: true,
+                    });
+                }
+
+                let response = `✅ Timed out \`${member.user.tag} (${member.id})\`.`;
+
+                if (!interaction.options.getBoolean('silent')) {
+                    const notified = await notifyUser(user, interaction, 'timed out');
+                    if (!notified) response += "\n\n⚠️ Couldn't send DM to user.";
+                }
+
+                try {
+                    await member.timeout(
+                        msDuration,
+                        interaction.options?.getString('reason') ? `${interaction.options?.getString('reason')}` : '',
+                    );
+                    return interaction.reply({ content: response, ephemeral: true });
+                } catch (error) {
+                    console.error(error);
+                    return interaction.reply({
+                        content: `❌ I encountered an error while trying to time out \`${member.user.tag}\`: \n\`\`\`${
+                            error?.message || error
+                        }\`\`\``,
+                        ephemeral: true,
+                    });
+                }
+            }
+            case 'remove': {
+                if (!member.isCommunicationDisabled()) {
+                    return interaction.reply({ content: '❌ This user is not currently timed out.', ephemeral: true });
+                }
+
+                if (member.moderatable === false) {
+                    return interaction.reply({
+                        content:
+                            '❌ I cannot remove the timeout from this user! \n**Please make sure that my highest role is above theirs.**',
+                        ephemeral: true,
+                    });
+                }
+
+                const response = `✅ Removed the timeout from \`${member.user.tag} (${member.id})\``;
```

Comment:
Since `timeout` is preceded by a `the` and a user can only have one timeout applied, I feel like using `for` is more natural sounding & grammatically correct than `from`.

```suggestion
                const response = `✅ Removed the timeout for \`${member.user.tag} (${member.id})\``;
```